src/cruds/remove.js

```diff
@@ -0,0 +1,18 @@
+function removeItem(itemIndex, data) {
+  // const item = document.getElementById(index);
+  //   item.remove();
+  //   data = data.filter((list) => list.index !== index);
+  //   localStorage.setItem('NOTE',JSON.stringify(data));
```

Comment:
- [ ] Please kindly remove `commented code` from your `code base`, as it makes your `code base` dirty and less readable. To make sure that your codebase is always clean, always remember to remove commented-codes.